libs/sdk-ui-kit/src/Header/HeaderUpsellButton.tsx

```diff
@@ -0,0 +1,19 @@
+// (C) 2021 GoodData Corporation
+import React from "react";
+import { FormattedMessage, injectIntl, WrappedComponentProps } from "react-intl";
+import { IHeaderUpsellButtonProps } from "./typings";
+
+export const CoreHeaderUpsellButton: React.FC<IHeaderUpsellButtonProps & WrappedComponentProps> = ({
+    onUpsellButtonClick,
+}) => {
+    return (
+        <button className="gd-button-small gd-button-primary gd-upsell-button" onClick={onUpsellButtonClick}>
+            <i className="gd-icon-star" />
+            <span className="gd-upgrade-button-text">
+                <FormattedMessage id="gs.header.upsellButtonText" />
+            </span>
+        </button>
+    );
+};
+
+export const HeaderUpsellButton = injectIntl(CoreHeaderUpsellButton);
```

Comment:
For functional components, I think that we should favor `useIntl()` hook instead of `injectIntl()` hoc. (Less code, less components and components props are not polluted with HOC props)

Reply:
I updated this component to use useIntl(), thanks!

Reply:
Thanks :)